packages/happy-dom/src/fetch/IRequest.ts

```diff
@@ -1,5 +1,8 @@
 import IHeaders from './IHeaders';
 import IBody from './IBody';
+import URL from './../location/URL';
+
+export type RequestInfo = IRequest | string | URL;
```

Comment:
I would prefer to put the type as an interface with "I" as prefix in its own file, so that it is easy to find it (e.g. by doing `ctrl + p` and search for it).

Reply:
ok, just that I understand you right, I will create a new file `IRequestInfo.ts` with only these two lines:
```typescript
import URL from './../location/URL';
export type IRequestInfo = IRequest | string | URL;
```